composer.json

```diff
@@ -26,6 +26,7 @@
         "ext-json": "*",
         "friendsofphp/php-cs-fixer": "^2.15",
         "phpunit/phpunit": "^8.4",
+        "symfony/console": "^4.1",
```

Comment:
This should be removed. It's already required above.

Reply:
I was wondering if maybe the integration tests need a higher version, since it was added with #8